tests/ballerina-test-integration/src/test/java/org/ballerinalang/test/observability/metrics/MetricsTestCase.java

```diff
@@ -0,0 +1,202 @@
+        expectedMetrics.put("http_response_time_seconds_max{http_method=\"GET\",http_url=\"/product\"," +
+                "protocol=\"http\",resource=\"getProduct\",service=\"metricsTest\",}", regex);
+        expectedMetrics.put("http_response_time_seconds{http_method=\"GET\",http_url=\"/test\",protocol=\"http\"," +
+                "resource=\"test\",service=\"metricsTest\",quantile=\"0.5\",}", regex);
+        expectedMetrics.put("http_response_time_seconds{http_method=\"GET\",http_url=\"/test\",protocol=\"http\"," +
+                "resource=\"test\",service=\"metricsTest\",quantile=\"0.75\",}", regex);
+        expectedMetrics.put("http_response_time_seconds{http_method=\"GET\",http_url=\"/test\",protocol=\"http\"," +
+                "resource=\"test\",service=\"metricsTest\",quantile=\"0.98\",}", regex);
+        expectedMetrics.put("http_response_time_seconds{http_method=\"GET\",http_url=\"/test\",protocol=\"http\"," +
+                "resource=\"test\",service=\"metricsTest\",quantile=\"0.99\",}", regex);
+        expectedMetrics.put("http_response_time_seconds{http_method=\"GET\",http_url=\"/test\",protocol=\"http\"," +
+                "resource=\"test\",service=\"metricsTest\",quantile=\"0.999\",}", regex);
+        expectedMetrics.put("http_response_time_seconds_count{http_method=\"GET\",http_url=\"/test\"," +
+                "protocol=\"http\",resource=\"test\",service=\"metricsTest\",}", Pattern.compile("6.0"));
+        expectedMetrics.put("http_response_time_seconds_sum{http_method=\"GET\",http_url=\"/test\"," +
+                "protocol=\"http\",resource=\"test\",service=\"metricsTest\",}", regex);
+        expectedMetrics.put("http_response_time_seconds{http_method=\"GET\",http_url=\"/product\",protocol=\"http\"," +
+                "resource=\"getProduct\",service=\"metricsTest\",quantile=\"0.5\",}", regex);
+        expectedMetrics.put("http_response_time_seconds{http_method=\"GET\",http_url=\"/product\",protocol=\"http\"," +
+                "resource=\"getProduct\",service=\"metricsTest\",quantile=\"0.75\",}", regex);
+        expectedMetrics.put("http_response_time_seconds{http_method=\"GET\",http_url=\"/product\",protocol=\"http\"," +
+                "resource=\"getProduct\",service=\"metricsTest\",quantile=\"0.98\",}", regex);
+        expectedMetrics.put("http_response_time_seconds{http_method=\"GET\",http_url=\"/product\",protocol=\"http\"," +
+                "resource=\"getProduct\",service=\"metricsTest\",quantile=\"0.99\",}", regex);
+        expectedMetrics.put("http_response_time_seconds{http_method=\"GET\",http_url=\"/product\",protocol=\"http\"," +
+                "resource=\"getProduct\",service=\"metricsTest\",quantile=\"0.999\",}", regex);
+        expectedMetrics.put("http_response_time_seconds_count{http_method=\"GET\",http_url=\"/product\"," +
+                "protocol=\"http\",resource=\"getProduct\",service=\"metricsTest\",}", Pattern.compile("6.0"));
```

Comment:
http_response_time_seconds_count is that is the response time in seconds?  if it's time, how can you be certain that is 6.0seconds always? What is the this metric actually represents?

Reply:
It is representing the count of recorded values(response time)